lib/tutorials/en_US/getting-started.md

```diff
@@ -45,35 +51,35 @@ Now that we have a server we should add one or two routes so that it actually do
 
 const Hapi = require('hapi');
 
-const server = new Hapi.Server();
-server.connection({ port: 3000, host: 'localhost' });
-
-server.route({
-    method: 'GET',
-    path: '/',
-    handler: function (request, reply) {
-        reply('Hello, world!');
-    }
-});
+const server = Hapi.server({ port: 3000, host: 'localhost' });
 
 server.route({
     method: 'GET',
     path: '/{name}',
-    handler: function (request, reply) {
-        reply('Hello, ' + encodeURIComponent(request.params.name) + '!');
+    handler: (request, h) => {
+
+        const name = request.params.name;
+        return `Hello, ${name ? name : 'world'}!`;
     }
 });
 
-server.start((err) => {
+const init = async () => {
 
-    if (err) {
-        throw err;
-    }
+    await server.start();
     console.log(`Server running at: ${server.info.uri}`);
+};
+
+process.on('uncaughtException', (err) => {
+
+    console.log(err);
+    process.exit(1);
 });
+
+init();
+
 ```
 
-Save the above as `server.js` and start the server with the command `node server.js`. Now you'll find that if you visit http://localhost:3000 in your browser, you'll see the text `Hello, world!`, and if you visit http://localhost:3000/stimpy you'll see `Hello, stimpy!`.
+Save the above as `server.js` and start the server with the command `node server.js`. Now you'll find that if you visit [http://localhost:3000](http://localhost:3000) in your browser, you'll see the text `Hello, world!`, and if you visit [http://localhost:3000/stimpy](http://localhost:3000/stimpy) you'll see `Hello, stimpy!`.
```

Comment:
This still doesn't work. There's no route configured now to handle the request to `/`. I would just put it back to how it was before to be honest.